frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -63,18 +70,26 @@ const Campaigns = () => {
     setLoading(false)
   }
 
+  function processAnnouncementVersion(announcementVersion: number) {
+    if (announcementVersion < +ANNOUNCEMENT.version) {
```

Comment:
if you make the change on backend to change the type to a string representing the package version in semver format, then this check has to be changed

Reply:
Made the appropriate type changes in the backend and frontend - now we need a way to compare the version numbers. I saw that `semver` has an npm package. Should we just install and use that, or should we write a parsing util?